src/modules/portfolio/containers/filled-orders.js

```diff
@@ -35,21 +35,34 @@ const mapStateToProps = state => {
     filledOrders: selectFilledOrders(
       groupedFilledOrders[item.id],
       account,
-      outcomesData,
-      marketsData
+      outcomesData[item.id],
+      marketsData[item.id]
     )
   }));
 
   /* eslint-disable */
   let allFilledOrders = [];
   marketIds.map(marketId => {
-    const formattedFilledOrders = selectFilledOrders(marketTradingHistory[marketId], account, outcomesData, marketsData);
+    const formattedFilledOrders = selectFilledOrders(marketTradingHistory[marketId], account, outcomesData[marketId], marketsData[marketId]);
```

Comment:
if we have marketDescription and outcomeName then would outcomesData and marketsData not be needed?